### PR TITLE
Docs(PPP): Add Azeem's PPP

### LINKED ISSUE #186: PPP: Azeem

### PR DESCRIPTION
**Pull request by DrWala**
## What this does
Closes #186

## How to test
1. cd to `docs/` folder of the project.
2. Run `bundle exec jekyll serve`.
3. Open the jekyll-hosted website in your browser (default address: http://127.0.0.1:4000).
4. Check that the About Us page has the changes listed above

### CODE REVIEW

File: docs/team/azeem.md [hunk 1]
```diff
@@ -5,42 +5,46 @@ title: Azeem's Project Portfolio Page
 
 ## Project: AddressBook Level 3
```
Comment (colintkn) [inline docs/team/azeem.md:6]: Can change to SunRez

File: docs/team/azeem.md [hunk 2]
```diff
@@ -5,42 +5,46 @@ title: Azeem's Project Portfolio Page
...
 
 Given below are my contributions to the project.
 
-* **New Feature**: Added the ability to undo/redo previous commands.
-  * What it does: allows the user to undo all previous commands one at a time. Preceding undo commands can be reversed by using the redo command.
-  * Justification: This feature improves the product significantly because a user can make mistakes in commands and the app should provide a convenient way to rectify them.
-  * Highlights: This enhancement affects existing commands and commands to be added in future. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required changes to existing commands.
-  * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+* **New Feature**: Added the ability to manage rooms (CRUD Rooms)
+  * What it does: Allows the user to manage the rooms of the hostel/RC
```
Comment (colintkn) [inline docs/team/azeem.md:13]: Final detail. But maybe Can add full stop here.

Comment (DrWala) [reply docs/team/azeem.md]: Done

File: docs/team/azeem.md (hunk 1 above)
Comment (DrWala) [reply docs/team/azeem.md]: Good catch thank you

**Review by benedictkhoomw — CHANGES_REQUESTED** [review]
The link from About Us to your PPP seems to be broken.

**Review by benedictkhoomw — APPROVED** [review]
Link works now. LGTM.

**Pull request merged**